db/migrate/20170419155310_rename_indices_for_oracle.rb

```diff
@@ -0,0 +1,10 @@
+class RenameIndicesForOracle < ActiveRecord::Migration
+  def change
+    unless NUCore::Database.oracle?
+      rename_index :secure_rooms_card_readers, "index_secure_rooms_card_readers_on_product_id", "index_card_readers_on_prod_id"
```

Comment:
These are fine, but I sometimes like to go with `i_` as a prefix when you need to cut out a few letters.